code/obj/flock/structure/sapper.dm

```diff
@@ -0,0 +1,147 @@
+#define CHARGING_BITS "Speed up Flockbits"
+#define CHARGING_DRONES "Charge incapacitors"
+#define CHARGING_STRUCTURES "Enhance structures"
+
+/obj/flock_structure/sapper
+	icon_state = "sapper-off" // current sprites are placeholders
```

Comment:
are they still placeholders (it's fine just checking if comment is relevant)

Reply:
Yeah, sprite is tracked in GitHub Flock project too